main/webapp/modules/core/scripts/dialogs/clustering-dialog.js

```diff
@@ -293,24 +307,32 @@ ClusteringDialog.prototype._renderTable = function(clusters) {
             ];
             var url = "project?" + params.join("&");
 
-            var div = document.createElement('div');
-            div.class = "clustering-dialog-value-focus";
-
             var browseLink = $(browseLinkTemplate).clone()
                 .attr("href",url)
                 .appendTo(div);
 
-            $(tr.insertCell(2))
+            var editCheck = $('<input type="checkbox" style = "accent-color: gray;" />')
+                .on('change',function() {
+                    cluster.edit = this.checked;
+                    const checkboxChoicesList = document.getElementsByClassName("Checkbox_Choice" + index.toString());
```

Comment:
I would generally try to use the jQuery API everywhere (instead of native DOM manipulation methods) to avoid the problem of having to mentally keep track whether a variable holds a native DOM element or a jQuery wrapper for it, and using consistent manipulation methods everywhere. I realize that this file was already a mix of jQuery and native methods before your changes, but I'd say you can stick to jQuery for the new things you introduce.